src/sentry/utils/snuba.py

```diff
@@ -1007,13 +1010,11 @@ def resolve_condition(cond, column_resolver):
     column_resolver (Function[string]) Function to resolve column names for the
                                        current dataset.
     """
+    # import pdb; pdb.set_trace()
     index = get_function_index(cond)
-    if index is not None:
-        # IN conditions are detected as a function but aren't really.
-        if cond[index] == "IN":
-            cond[0] = column_resolver(cond[0])
-            return cond
-        elif cond[index] in FUNCTION_TO_OPERATOR:
+    # IN/NOT IN conditions are detected as a function but aren't really.
+    if index is not None and cond[index] not in ("IN", "NOT IN"):
```

Comment:
We want to let this fall through so we can take advantage of the more advanced resolving that happens later in this function.